### PR TITLE
Newsweek page Clone

### PR DESCRIPTION
**Pull request by ZawadiSumba66**
Hi TSE, 👋

This is a clone of Newsweek. It was built using Bootstrap technique and Bem naming for CSS and also it is fully responsive.

Best,

Murodjon and Rose🙂

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+## Built With
+
+- HTML / CSS
+- BOOTSTRAP 4
+
+## Live Demo
+
+[Live Demo Link](https://murodjon000.github.io/newsweek-clone/)
+
```
Comment (arslanbisharat) [inline README.md:18]: - [ ] [Optional] You can add heading to this line  "Setup"

File: README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+## Authors
+
+👤 **Murodjon000**
+
+- GitHub: [@Murodjon 0103](https://github.com/Murodjon000)
+
+👤 **ZawadiSumba66**
+
+- GitHub: [@ZawadiSumba66](https://github.com/ZawadiSumba66)
```
Comment (arslanbisharat) [inline README.md]: - [ ] [Optional] You can add more social links here

File: styles/backgroun.css [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+@media only screen and (min-width: 768px) {
+  .md\:full-white {
+    background-color: white;
+  }
+}
+
+.bg-primary {
+  background-color: #f72210 !important;
```
Comment (arslanbisharat) [inline styles/backgroun.css]: - [ ] Please remove !important

File: styles/body.css [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+.category > span {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  padding: 8px 15px 9px;
+  background-color: rgba(255, 255, 255, 0.9);
+  z-index: 1;
+  text-decoration: none !important;
```
Comment (arslanbisharat) [inline styles/body.css]: - [ ] Please reomve !important

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Request changes 😳

I am very happy to see this project and I can see you worked hard on this project. There are only a few issues that we need to address and I am sure that you will end up with a perfect project

_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_

This is a great opportunity to learn. Microverse is here to help you. 

**Please have a look at the suggestions to get more understanding.**

- [x] The main purpose of this project is to use Bootstrap and not actual plain CSS. But have used too much CSS. Please work on your bootstrap skills and change CSS to bootstrap. 

- [x] Please don't use important in CSS. This is not a good practice in writing clean code. 
- [x] Please remove commented Code. 
- [x] Please add all unnessacry files into .gitignore. 


- [x] Please add styling folders in a single folder. 


- [x] Please add the proper heading to these sections and the Newsletter section should have equal height as the Subscribe section. 





**Please make the changes and request a re-review for the given project**

If you get stuck somewhere, don't hesitate to reach me out or schedule s 1:1 meeting with a Full-time TSE. 

Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by arslanbisharat — COMMENTED** [review]
# Invalid Submission

This project is marked invalid because it's not professional. 

**Friendly reminder:** 

You should make sure that your repo looks professional before review.
Please take a step back and read a lesson [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/).

Then apply 3 simple rules from this lesson (it will be easy as you can use a template provided there).

This Code Review Request is marked as invalid in your Dashboard, so please submit a new one once you are ready!

If you get stuck somewhere, don't hesitate to reach me out or schedule s 1:1 meeting with Full-time TSE. 

Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,

Arslan Bisharat

slack: arslanbisharat**

**Comment by mimipeshy** [comment]
## STATUS: CHANGES REQUIRED 🔴 ♻️ 

Hello @ZawadiSumba66 and @Murodjon000  👋 👋 , I must say I am impressed by your work. Your website looks like the original one 👏 👏 .
✔️ Good job on working on the TSE requirements
 However, before marking the project as  complete, let's fix some things

- [ ]  On the mobile menu, the featured stories should appear right below the top story.  This is followed by the debate section, more stories at last my turn, and the culture and travel section. Check on this 😄 . You can refer to this [link](https://www.newsweek.com/).

[OPTIONAL]

- [ ] You can choose to host your website on a server eg [gh-pages](https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/creating-a-github-pages-site). Its pretty easy to do so, then add the live link to the README file. This makes it easy for people to see your projects 😄 


That's all. 😄 , kindy submit another review when you are done

Regards
Peris Ndanu
Feel free to reach out on slack if you have any questions

**Review by jubaan — APPROVED** [review]
# :tada: :tada: :tada: :rotating_light: Status: Approved WITH NOTES:rotating_light:  :tada: :tada: :tada:

The project is approved and it looks awesome.

I think you have done a great job so far and the website looks incredible. There are some little observations that might help the clone look even more awesome. While this is entirely optional I encourage you to fix them, I'm sure they will not take too much of your time and it will be worth it.

These sections are not looking like the original in some aspects please compare with original to notice the differences.
image
 
## See you at the top :rocket:!

# :tada: :tada: :tada: :rotating_light: Status: Approved :rotating_light:  :tada: :tada: :tada:

**Pull request merged**